sw/device/tests/meson.build

```diff
@@ -691,6 +691,26 @@ sw_tests += {
   }
 }
 
+flash_ctrl_clock_freqs_test_lib = declare_dependency(
```

Comment:
Can you also add bazel target to BUILD file?
you can see an example here(https://github.com/lowRISC/opentitan/pull/12188/files#diff-923e565356e32daee8f8bbe454109866398ebb25664edfa81a9f6aede76eb297)